ssl/t1_lib.c

```diff
@@ -1466,6 +1703,13 @@ static int sigalg_security_bits(SSL_CTX *ctx, const SIGALG_LOOKUP *lu)
         else if (lu->sigalg == TLSEXT_SIGALG_ed448)
             secbits = 224;
     }
+    /* For provider-based sigalgs we have secbits information available
+     * in the (provider-loaded) sigalg_list structure
+     */
+    if ((secbits == 0) && (lu->sig_idx >= SSL_PKEY_NUM)
+               && ((lu->sig_idx - SSL_PKEY_NUM) < (int)ctx->sigalg_list_len)) {
+        secbits = ctx->sigalg_list[lu->sig_idx - SSL_PKEY_NUM].secbits;
+    }
```

Comment:
In the above code the security bits is based purely on the digest if one is available. Presumably that is always correct for the current sigalgs. I wonder if it is always true for provider based sigalgs which include a digest

Reply:
See documentation update mandating sigalg providers to set secbits suitably for a combined algorithm.

Reply:
@mattcaswell, please let me know if there's something I can do to get this conversation resolved.

Reply:
Let's leave it for now. Probably this is an issue not directly related to the main intent of this PR.